aidants_connect_habilitation/static/scss/main-habilitation.scss

```
@@ -99,7 +99,7 @@ h1, .h1 {
   line-height: 1.25;
 }
 
-h1, .h1, .subtitle-container {
+h1, .h1, h2, .h2 .subtitle-container {
```

Avant on ciblait les `.subtitle-container` et maintenant on ne vise plus que les `.subtitle-container` à l'intérieur d'un `.h2`… Je suppose que tu voulais plutôt ceci, non ?

```suggestion
h1, .h1, h2, .h2, .subtitle-container {
```